requirements.txt

```diff
@@ -1,8 +1,8 @@
 hydra-core
 omegaconf
-pytorch-lightning~=1.3.8
+pytorch-lightning
```

Comment:
Pytorch lightning can develop rapidly, I think limiting the version to be between 1.3 and 1.4 is a safer requirement. Something like `pytorch-lightning>=1.3,<1.5` should do this.